README.md

```diff
@@ -22,7 +22,12 @@ au User asyncomplete_setup call asyncomplete#register_source(asyncomplete#source
     \ 'whitelist': ['javascript'],
     \ 'completor': function('asyncomplete#sources#flow#completor'),
     \ 'config': {
-    \    'flowbin_path': expand('~/bin/flow')
+    \    " Resolves 'flow' in the closest node_modules/.bin directory (in case
+    \    " flow is installed via 'npm install flow-bin' locally). Falls back to
+    \    " 'flowbin_path' (see below) if can't find it.
+    \    'resolve_flowbin_path': 1
```

Comment:
Might be rename to `prefer_local` that way even other sources can use the same key.